i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md

```diff
@@ -3,3 +3,29 @@ sidebar_position: 5
 ---
 
 # 🪛 配置连接器
+
+_连接器_ 是 Logto 和那些提供短信邮件的服务商以及提供用户信息的社交媒体之间的桥梁。
+
+Logto 提供了很多能让用户可以访问三方服务提供商的连接器，包括了 Google，GitHub，Facebook，SendGrid 等等。
+
+当你想要使用 _连接器_ 的时候可能会感到迷茫，毕竟他是一个全新的概念。:sob:
+
+但是不要悲伤不要心急，看完这篇文档可以让你轻松拿捏！ :pinching_hand:
+
+## :watch: 开始上手 :zap:
+
+让我们先从管理控制台开始吧！
+
+首先需要登录 Logto 帐号（这需要你的账号有管理员权限）。如果没有账号或者没有管理员权限的话，你需要先创建一个，详见「[创建一个管理员账户](../../tutorials/get-started/README.mdx#创建一个管理员账户)」。
+
+点击左侧侧边栏中的「连接器」按钮 :neutral_face:
+
+:ghost: **欢迎来到 _连接器_ 的世界** :ghost:
+
```

Comment:
do we have a figure for this? if so would you like to add some comments

Reply:
We don't have the figure for this. Added some brief explanations to make the transition more smooth.